package.json

```diff
@@ -187,17 +189,17 @@
     "@zeit/dockerignore": "0.0.3",
     "JSONStream": "^1.0.3",
     "archiver": "^3.1.1",
-    "balena-config-json": "^4.0.0",
+    "balena-config-json": "^4.1.0",
     "balena-device-init": "^5.0.2",
-    "balena-errors": "^4.3.0",
-    "balena-image-manager": "^7.0.1",
-    "balena-preload": "^10.2.0",
+    "balena-errors": "^4.4.1",
+    "balena-image-manager": "^7.0.3",
+    "balena-preload": "^10.2.4",
     "balena-release": "^3.0.0",
-    "balena-sdk": "^14.8.0",
-    "balena-semver": "^2.2.0",
+    "balena-sdk": "^15.2.3",
```

Comment:
Not by default, it would only cause it in the case that a submodule updates balena-sdk and we update that submodule without also updating the top level balena-sdk and/or running npm dedupe.